docs/eventing/samples/kafka/channel/README.md

```diff
@@ -220,29 +239,87 @@ $ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
   --from-literal=user="my-sasl-user"
 ```
 
-*NOTE:* It is important to use the same keys; `user`, `password` and `saslType`.
-
-### SASL authentication using public CA certificates
-
-If you want to use SASL with public CA certificates, you must use the `tls.enabled=true` flag, rather than the `ca.crt` argument, when creating the secret. For example:
+or, if you want to use public CA certificates, you must use the
+`tls.enabled=true` flag, rather than the `ca.crt` argument, for example:
 
 ```
 $ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
-    --from-literal=tls.enabled=true \
+  --from-literal=tls.enabled=true \
   --from-literal=password="SecretPassword" \
   --from-literal=saslType="SCRAM-SHA-512" \
   --from-literal=user="my-sasl-user"
 ```
 
-Reference your secret and the namespace of the secret in the `config-kafka` ConfigMap:
+*NOTE:* It is important to use the same keys; `user`, `password` and `saslType`.
```

Comment:
```suggestion
!!! note
    It is important to use the same keys; `user`, `password` and `saslType`.
```
There's new formatting to use admonitions in mkdocs: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage

Reply:
I have replaced this and the other collection of old *NOTE:* formatting in the file with the new style.